runtime/browser-demo/browser-demo.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * @license
+ * Copyright (c) 2017 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+//let runtime = require("../runtime.js");
+
+let Arc = require("../arc.js");
```

Comment:
I'm curios, do we really need both - browser-test and browser-demo?
The test doesn't really fail on anything, so it was more of a demo already.

That said, what you added here (the suggestions) overlap a bit with what I was working on, so I can just unify both in my upcoming pull request, if that sounds reasonable.